hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OFSPath.java

```diff
@@ -78,8 +88,8 @@ private void initOFSPath(String pathStr) {
     // pathStr should not have authority
     try {
       URI uri = new URI(pathStr);
-      String authority = uri.getAuthority();
-      if (authority != null && !authority.isEmpty()) {
+      String auth = uri.getAuthority();
```

Comment:
Thinking more, do we need this, from above uri, we can get authority and check if it is empty or nulll and then throw error. So, in this way, we don't need to convert path to URI again.